package.json

```diff
@@ -1,37 +1,40 @@
 {
-  "name": "laravel-mix-purgecss",
-  "version": "1.0.5",
-  "description": "Purgecss wrapper for Laravel Mix",
-  "main": "src/index.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/spatie/laravel-mix-purgecss.git"
-  },
-  "keywords": [
-    "spatie"
-  ],
-  "author": "Sebastian De Deyne",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/spatie/laravel-mix-purgecss/issues"
-  },
-  "homepage": "https://github.com/spatie/laravel-mix-purgecss",
-  "scripts": {
-    "format": "prettier --write \"**/*.js\""
-  },
-  "dependencies": {
-    "glob-all": "^3.0.0",
-    "lodash": "^4.0.0",
-    "purgecss-webpack-plugin": "^0.19.0"
-  },
-  "devDependencies": {
-    "laravel-mix": "^1.7.0",
-    "prettier": "^1.10.1"
-  },
-  "peerDependencies": {
-    "laravel-mix": "^2.0.0"
-  },
-  "engines": {
-    "node": ">=6.0.0"
-  }
+    "name": "laravel-mix-purgecss",
```

Comment:
Could you revert this indentation change?

Reply:
Done